worker/host.go

```diff
@@ -22,7 +22,7 @@ type (
 		UploadSector(ctx context.Context, sector *[rhpv2.SectorSize]byte, rev types.FileContractRevision) (types.Hash256, error)
```

Comment:
I think this is missing a few spots. Search for `rhpv3.PayByEphemeralAccount(` I found at least 2 more.

- In `preparePriceTableAccountPayment` 
- In `fetchRevisionWithAccount`